sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q17.sf100/simplified.txt

```diff
@@ -1,97 +1,90 @@
 TakeOrderedAndProject [i_item_id,i_item_desc,s_state,store_sales_quantitycount,store_sales_quantityave,store_sales_quantitystdev,store_sales_quantitycov,as_store_returns_quantitycount,as_store_returns_quantityave,as_store_returns_quantitystdev,store_returns_quantitycov,catalog_sales_quantitycount,catalog_sales_quantityave,catalog_sales_quantitystdev,catalog_sales_quantitycov]
-  WholeStageCodegen (18)
+  WholeStageCodegen (16)
```

Comment:
can you highlight the reason of the plan changes?

Reply:
Since we already have `Exchange [ss_item_sk] #4`, the shuffle:

`Exchange [ss_customer_sk,ss_item_sk,ss_ticket_number] #3` can be saved since it includes `ss_item_sk`.
And shuffle `Exchange [sr_customer_sk,sr_item_sk] #2` can also be saved since `sr_item_sk` is equal to `ss_item_sk`.